src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -40,8 +42,10 @@ public AddCommand parse(String args) throws ParseException {
         Quantity quantity = ParserUtil.parseQuantity(argMultimap.getValue(PREFIX_QUANTITY).get());
         Supplier supplier = ParserUtil.parseSupplier(argMultimap.getValue(PREFIX_SUPPLIER).orElse("No Supplier"));
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Quantity maxQuantity = ParserUtil.parseMaxQuantity(argMultimap.getValue(PREFIX_MAX_QUANTITY)
+                .orElse("0"));
```

Comment:
Similarly to tag i think initiazing to a 0 is not a good idea because we'll eventually be using it to calculate percentages

Reply:
I have refactored maxQuantity to use Optionals instead of using "0" now